src/classes/list-object.ts

```diff
@@ -0,0 +1,34 @@
+import { NS_RDF } from '../constants'
```

Comment:
For consistency with the generator and components can we make this an alias of `@constants`?